deployments/boba.json

```
@@ -28,5 +28,8 @@
   "LiFiDEXAggregator": "0x6140b987d6B51Fd75b66C3B07733Beb5167c42fc",
   "EmergencyPauseFacet": "0x6F2baA7cd5F156CA1B132F7FF11E0fa2aD775F61",
   "RelayFacet": "0x424BDbbaEda89732443fb1B737b6Dc194a6Ddbd5",
-  "Permit2Proxy": "0x89c6340B1a1f4b25D36cd8B063D49045caF3f818"
-}
\ No newline at end of file
+  "Permit2Proxy": "0x89c6340B1a1f4b25D36cd8B063D49045caF3f818",
```

Biome (1.9.4)

[error] The key Permit2Proxy was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)